styles.css

```diff
@@ -426,3 +427,424 @@ p {
+    opacity: 0.5;
+  }
+
+  .social-media {
+    margin: 0;
+    padding-top: 10px;
+    display: flex;
+    justify-content: flex-start;
+    gap: 20px;
+    list-style: none;
+    font-size: 20px;
+  }
+
+  /* cards */
+  .work-section {
+    padding: 20px;
+    margin: 0 auto;
+    padding-left: 90px;
+    padding-right: 90px;
+  }
+
+  .work-container {
+    background-color: #fff;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    padding-top: 50px;
+    align-items: center;
+    border-radius: 20px;
+    margin: 50px auto;
+    border: 1px solid #e6e8eb;
+  }
+
+  .card {
+    width: 100%;
+  }
+
+  .left-block {
+    height: 35vw;
+    width: 50%;
+    flex: 2;
+    padding: 20px;
+    align-items: center;
+    margin: 0 auto;
+  }
+
+  .reverse {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .left-block h2 {
+    font-size: 38px;
+    line-height: 44px;
+  }
+
+  .position {
+    display: flex;
+    margin-top: 10px;
+    gap: 15px;
+  }
+
+  .position li {
+    font-weight: 590;
+    font-size: 11.5px;
+    color: #7a869a;
+    list-style: none;
+  }
+
+  .languages i {
+    padding-right: 35px;
+  }
+
+  .position li i {
+    color: #7a869a;
+    font-size: 10px;
+  }
+
+  .programming-lang li {
+    flex-wrap: wrap;
+    margin-bottom: 10px;
+    list-style: none;
+  }
+
+  .frames li {
+    color: #000;
+    list-style: none;
+    font-size: 13px;
+    font-weight: 500px;
+    line-height: 24px;
+  }
+
+  .language-list li {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+  }
+
+  .form li {
+    margin-top: 1.3rem;
+    list-style: none;
+    border-radius: 5px;
+  }
+
+  .programming-lang ul li {
+    padding-right: 10px;
+  }
+
+  .position li:first-child {
+    color: #344563;
+    list-style: none;
+  }
+
+  .programming-lang {
+    display: flex;
+    margin-top: 0;
+  }
+
+  .work-container p {
+    font-weight: 400;
+    font-size: 15px;
+    color: #344563;
+  }
+
+  .btn-action {
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    width: 126px;
+    height: 48px;
+    background: #fff;
+    font-weight: 500;
+    font-size: 17px;
+    margin-top: 10px;
+    color: #396df2;
+  }
+
+  .btn-action:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  .btn-action:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .about-myself {
+    display: flex;
+    flex-direction: row;
+    margin-top: 50px;
+    padding: 100px;
+    padding-top: 40px;
+    background-color: #fff;
+    border-top-right-radius: 120px;
+  }
+
+  .about-container {
+    flex: 1;
+    margin-top: 70px;
+    width: 40%;
+  }
+
+  .about-myself p {
+    color: #344563;
+    font-size: 15px;
+    font-weight: 400px;
+  }
+
+  .skill-container {
+    width: 100%;
+  }
+
+  .language-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    gap: 20px;
+  }
+
+  .languages {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
I could see repeated usage of `display: flex` in your styling, to follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Please let's reduce code repetition, If we want to create a CSS rule for multiple elements, we can use a [class attribute instead](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 😀